addons/l10n_ro/models/account_move.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    l10n_ro_is_self_invoice = fields.Boolean(
+        string='Is self invoice?',
+        help='Check this box to indicate that this bill is a self-invoice made to register '
+             'a VAT event in the absence of an invoice received from a supplier.',
+    )
```

Comment:
I think I discussed those kind of invoice with TSB a while ago.
Don't we have some standard stuff for that?

Reply:
Do we?

Reply:
I guess that in the end the answer from Benj was that we don't have any standard thing for that...

Reply:
Ideally, we would see if we have some tax related to certain lines in the tax report, but this might be the simplest for now.

Reply:
@antoine162 @Levizar  I am not against having the field to go forward right now, but you can not add it in stable in the module in 16.  An option is to simply move it to l10n_ro_saft for now.